README.md

```diff
@@ -29,27 +29,27 @@ and renders declaratively using `lit-html`.
 
     Property options include:
 
-    * `attribute`: Describes how and whether the property becomes an observed attribute.
+    * `attribute`: Indicates how and whether the property becomes an observed attribute.
     If the value is `false`, the property is not added to `observedAttributes`.
     If true or absent, the lowercased property name is observed (e.g. `fooBar` becomes `foobar`).
     If a string, the string value is observed (e.g `attribute: 'foo-bar'`).
-    * `type`: Describes how to serialize and deserialize the attribute to/from a property.
+    * `type`: Indicates how to serialize and deserialize the attribute to/from a property.
     If this value is a function, it is used to deserialize the attribute value
     a the property value. If it's an object, it can have keys for `fromAttribute` and
     `toAttribute` where `fromAttribute` is the deserialize function and `toAttribute`
     is a serialize function used to set the property to an attribute. If no `toAttribute`
     function is provided and `reflect` is set to `true`, the property value is set
     directly to the attribute.
-    * `reflect`: Describes if the property should reflect to an attribute.
+    * `reflect`: Indicates if the property should reflect to an attribute.
     If `true`, when the property is set, the attribute is set using the
     attribute name determined according to the rules for the `attribute`
     propety option and the value of the property serialized using the rules from
     the `type` property option.
-    * `shouldInvalidate`: Describes if setting a property should trigger
-    invalidation and updating. This function takes the `newValue` and `oldValue` and
-    returns `true` if invalidation should occur. If not present, a strict identity
-    check is used. This is useful if a property should be considered dirty only
-    if some condition is met, like if a key of an object value changes.
+    * `hasChanged`: Indicates if a property should be considered changed when it's set.
```

Comment:
The tense here makes it sound like it is a function you call, rather than a function you supply.  

Suggestion: 
>`hasChanged`: A function that indicates if a property should be considered changed when it is set. The function should take the `newValue` and `oldValue` and return `true` if an update should be requested.

Reply:
Done